core/platform/taskqueue/cloud_tasks_emulator.py

```diff
@@ -194,9 +208,10 @@ def create_task(
                 queue_name, url, payload, scheduled_for=scheduled_for_time,
                 task_name=task_name)
             queue.append(task)
-            queue.sort(key=lambda t: t.scheduled_for)
+            k = lambda t: t.scheduled_for
```

Comment:
Explain why is this neededm as you did in your other PR.

Reply:
Done